### PR TITLE
Make Default JobMasterClient.getJobStatus non-verbose

### PR DESCRIPTION
**Pull request by bradyoo**
Fixes https://github.com/Alluxio/alluxio/issues/11184

### CODE REVIEW

**Comment by bradyoo** [comment]
ping @gpang for review
ping @yuzhu because this at least addresses the issue of running a job crashing to GRPC data limit. (You still won't be able to run getStatus on it because that gets verbose)

**Comment by jinhang** [comment]
m

File: job/server/src/main/java/alluxio/master/job/JobMasterClientServiceHandler.java [hunk 1]
```diff
@@ -69,7 +71,18 @@ public void getJobStatus(GetJobStatusPRequest request,
...
+
+  @Override
+  public void getJobStatusDetailed(GetJobStatusDetailedPRequest request,
+                                   StreamObserver<GetJobStatusDetailedPResponse>
+                                       responseObserver) {
+    RpcUtils.call(LOG, (RpcUtils.RpcCallableThrowsIOException<GetJobStatusDetailedPResponse>) () ->
+    {
+      return GetJobStatusDetailedPResponse.newBuilder()
+          .setJobInfo(mJobMaster.getStatus(request.getJobId(), true).toProto()).build();
     }, "getJobStatus", "request=%s", responseObserver, request);
```
Comment (gpang) [inline job/server/src/main/java/alluxio/master/job/JobMasterClientServiceHandler.java]: this should be `getJobStatusDetailed`?

**Comment by bradyoo** [comment]
@gpang: This fixes all the sides that aren't. Job stat command. For example, a bunch of end points use JobGrpcClientUtils.

**Comment by bradyoo** [comment]
feedback addressed.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**